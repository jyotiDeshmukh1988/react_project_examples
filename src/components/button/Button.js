/* eslint-disable react/prop-types */
import React from "react";

export default function Button(props) {
  return <button>That is my btn {props.text}</button>;
}
