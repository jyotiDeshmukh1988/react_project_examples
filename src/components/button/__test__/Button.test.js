import { render, screen } from "@testing-library/react";
import renderer from 'react-test-renderer';
import Button from "../Button";
import React from 'react'

//test suite
describe("I am Testing util file", () => {
  it("i am testing f1 function", () => {
    const tree = renderer
      .create(<Button text={'mybtn'}></Button>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
