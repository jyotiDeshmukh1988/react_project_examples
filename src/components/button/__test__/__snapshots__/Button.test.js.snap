// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`I am Testing util file i am testing f1 function 1`] = `
<button>
  That is my btn 
  mybtn
</button>
`;
